sky/build/sdk_xcode_harness/FlutterApplication.xcodeproj/project.pbxproj

```diff
@@ -502,6 +502,7 @@
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				FLUTTER_APPLICATION_PATH = path/to/application;
+				FLUTTER_PATH = path/to/flutter/repo;
```

Comment:
We've been calling this the FLUTTER_ROOT to distinguish it from the path to the flutter package, which is inside the flutter root.